From dc35250b8f316df340d56e45e14d05dc084f7289 Mon Sep 17 00:00:00 2001
From: Michael Forney <mforney@mforney.org>
Date: Wed, 27 Nov 2019 19:37:17 -0800
Subject: [PATCH] Remove trailing semicolon after RB_GENERATE_STATIC

This expands to a series of function definitions, so the semicolon is
not necessary (in fact, it is not allowed in ISO C).
---
 krl.c | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/krl.c b/krl.c
index aa8318cf..01522b8f 100644
--- a/krl.c
+++ b/krl.c
@@ -60,7 +60,7 @@ struct revoked_serial {
 };
 static int serial_cmp(struct revoked_serial *a, struct revoked_serial *b);
 RB_HEAD(revoked_serial_tree, revoked_serial);
-RB_GENERATE_STATIC(revoked_serial_tree, revoked_serial, tree_entry, serial_cmp);
+RB_GENERATE_STATIC(revoked_serial_tree, revoked_serial, tree_entry, serial_cmp)
 
 /* Tree of key IDs */
 struct revoked_key_id {
@@ -69,7 +69,7 @@ struct revoked_key_id {
 };
 static int key_id_cmp(struct revoked_key_id *a, struct revoked_key_id *b);
 RB_HEAD(revoked_key_id_tree, revoked_key_id);
-RB_GENERATE_STATIC(revoked_key_id_tree, revoked_key_id, tree_entry, key_id_cmp);
+RB_GENERATE_STATIC(revoked_key_id_tree, revoked_key_id, tree_entry, key_id_cmp)
 
 /* Tree of blobs (used for keys and fingerprints) */
 struct revoked_blob {
@@ -79,7 +79,7 @@ struct revoked_blob {
 };
 static int blob_cmp(struct revoked_blob *a, struct revoked_blob *b);
 RB_HEAD(revoked_blob_tree, revoked_blob);
-RB_GENERATE_STATIC(revoked_blob_tree, revoked_blob, tree_entry, blob_cmp);
+RB_GENERATE_STATIC(revoked_blob_tree, revoked_blob, tree_entry, blob_cmp)
 
 /* Tracks revoked certs for a single CA */
 struct revoked_certs {
-- 
2.25.0

